log_outgoing_requests/admin.py

```diff
@@ -76,6 +78,7 @@ class OutgoingRequestsLogAdmin(admin.ModelAdmin):
         "response_ms",
         "res_headers",
         "res_content_type",
+        "response_content_length",
```

Comment:
I think it would be nice if it's possible to sort by this field in the admin listview, that way it's easier to find requests with unexpectedly large response bodies. Since it's a computed field, you might have to annotate the queryset to implement this: https://docs.djangoproject.com/en/4.2/ref/models/querysets/#annotate

Reply:
I'm absolutely  agree with making it sortable, but I think it is not possible to annotate the queryset because this property reads the length of another computed property `response_body_decoded`. I think it is therefore for this reason, I could be wrong.

Reply:
Hmm that's true, though I do wonder how useful this if we cannot sort by length.

I think it's only possible to make it sortable if we use `res_body` instead of `response_body_decoded`, although I do think the lengths won't be exactly correct depending on the content_type of the response.

Example for an HTML response
```
>>> len(large.res_body)
233210
>>> len(large.response_body_decoded)
233181
```

We could do something like this in that case:

```python
from django.db.models import Func

class LengthFunction(Func):
    function = 'LENGTH'


def get_queryset(self, request):
        qs = super().get_queryset(request)
        return qs.annotate(
            annotated_response_length=LengthFunction('response_body_decoded')
        )

def response_content_length(self, obj):
    return obj.annotated_response_length
response_content_length.admin_order_field = 'annotated_response_length'
```

@alextreme what are your thought on this. Would it be okay if the `response_content_length` in the list view isn't exactly correct, but it would be sortable? Or would you rather have the exact length without sortability?